### DISCUSSION TITLE
feature request: remove/improve automatic parent folder tagging

### DISCUSSION THREAD

**Discussion by jceb** [standardnotes/forum#2645] [Suggestions]
The 3.8 update brought automatic parent folder tagging which completely broke my workflow. I use the tag folder extension with a 3-4 level nested structure of tags. While before 3.8 I had usually just one tag for every note and used the nested structure to move notes from tag to tag easily. With 3.8 a change of tags automatically adds tags to the note for the whole structure. Suddenly it becomes a real hassle to move a note between tags, because I don't want the note to show up in the parent structure and therefore have to remove the added tags manually.

With the current system in place I don't get how nested tags should work. Already with a shallow tag structure of two levels, `#topic.a` and `#othertopic.b`, it becomes a real effort to move a note from `a` to `b`. I'd have to manually remove tags `#topic` and `#topic.a` before I can fully move the note over to `#othertopic.b`. The issue increases with increased nesting.

Another challenge is the visual clutter that's introduced by parent tagging. I think of tags as significant pieces of information when I scan my notes, However, with the increase amount of tags that are mostly irrelevant to me, it becomes difficult to find the relevant tag.

I'm not fully opposing the change's intention of listing notes in parent tags but couldn't this be achieved in a more subtle way? With tag `#topic.a` it can be assumed that the note is also part of the parent `#topic` tag. This way the visual clutter could be removed while providing similar functionality and unifying the experience of how notes are listed in tags.

**Comment by JaspalSuri** [comment]
Hi @jceb, sorry to hear that. Thanks for sharing your feedback on this with us. We'll continually monitor additional feedback on this behavior and adjust it accordingly as some users like this behavior. Hopefully we'll devise a solution that satisfies both preferences. 🙂

**Comment by myreli** [comment]
There is now a preference to disable automated tag nesting:

<img width="946" alt="image" src="https://user-images.githubusercontent.com/17554234/218859139-25a2cccb-88c6-48e8-861f-6202b769a0d3.png">
